src/CMakeLists.txt

```diff
@@ -669,32 +751,75 @@ if (${NUPIC_BUILD_PYEXT_MODULES})
   # Tell swig which command-line options to use, allowing user to override
   set(CMAKE_SWIG_FLAGS ${src_swig_flags} ${CMAKE_SWIG_FLAGS})
 
-  set(src_swig_support_files
```

Comment:
These are in merged into nupic_core from nupic_core_solo static lib, so no need to include them again.